addons/account/i18n/vi.po

```diff
@@ -62,17 +62,17 @@ msgstr " Loại %(lock_type)s có ngày khóa được cài ngày %(lock_date)s.
 #. module: account
 #: model:ir.model.fields,field_description:account.field_account_payment__reconciled_bills_count
 msgid "# Reconciled Bills"
-msgstr "# Đã được dối soát"
+msgstr "# Đã được Đối soát"
 
 #. module: account
 #: model:ir.model.fields,field_description:account.field_account_payment__reconciled_invoices_count
 msgid "# Reconciled Invoices"
-msgstr "# Hóa đơn đã đối soát"
+msgstr "# Hóa đơn Đã đối soát"
```

Comment:
Người Việt không có thói quen dùng `#` thay cho `số lượng` hay `số thứ tự` nên các chỗ `#` cần thay bằng `SL` hoặc `STT`, tùy ngữ cảnh.
```suggestion
msgstr "SL Hóa đơn Đã đối soát"
```
CC @Viindoo/product @Viindoo/project @Viindoo/is